lib/mimic/cover.ex

```diff
@@ -11,6 +11,18 @@ defmodule Mimic.Cover do
     :cover.is_compiled(module) != false
```

Comment:
this function here, would it make sense to rename it to `enabled_for_module?` or something like that? trying to figure out an appropriate name for it given the similarity in name with `cover_enabled?`